includes/class-wp-job-manager-post-types.php

```diff
@@ -247,7 +247,7 @@ public function register_post_types() {
 						'labels'               => [
 							'name'              => $plural,
 							'singular_name'     => $singular,
-							'menu_name'         => ucwords( $plural ),
+							'menu_name'         => __( 'Types', 'wp-job-manager' ),
```

Comment:
Should we also rename 'Job Categories' to 'Categories'? 
<img width="141" alt="image" src="https://github.com/Automattic/WP-Job-Manager/assets/176949/22b52d25-58cd-4427-a296-8eeee620669e">

Reply:
This is a good question. I renamed in 2ccdcab, but then we might want to update the Job Tags addon to use only "Tags" on the menu? Not sure.